octopuslite/reader.py

```diff
@@ -104,43 +95,48 @@ def __getitem__(self, channel_name: Union[str, Channels]):
 
         return self._lazy_arrays[channel_name]
 
-
     def files(self, channel_name: str) -> list:
         return self._files[Channels[channel_name.upper()]]
 
     def _load_and_process(self, fn: str) -> np.ndarray:
         """Load and crop the image."""
         image = io.imread(fn)
 
-        t = int(parse_filename(fn)['time'])
-        image = self._transformer(image, t)
+        if self._crop is not None:
 
-        if self._crop is None:
-            return image
+            assert isinstance(self._crop, tuple)
 
-        assert isinstance(self._crop, tuple)
+            dims = image.ndim
+            shape = image.shape
+            crop = np.array(self._crop).astype(np.int64)
 
-        dims = image.ndim
-        shape = image.shape
-        dtype = image.dtype
-        crop = np.array(self._crop).astype(np.int64)
+            # check that we don't exceed any dimensions
+            assert all([crop[i] <= s for i, s in enumerate(shape)])
 
-        # check that we don't exceed any dimensions
-        assert all([crop[i] <= s for i, s in enumerate(shape)])
+            # automagically build the slices for the array
+            cslice = lambda d: slice(
+                int((shape[d] - crop[d]) // 2), int((shape[d] - crop[d]) // 2 + crop[d])
+            )
+            crops = tuple([cslice(d) for d in range(dims)])
+            image = image[crops]
 
-        # automagically build the slices for the array
-        cslice = lambda d: slice(
-            int((shape[d] - crop[d]) // 2),
-            int((shape[d] - crop[d]) // 2 + crop[d])
-        )
-        crops = tuple([cslice(d) for d in range(dims)])
+        # check channel to see if label
+        channel = parse_filename(fn)["channel"]
+        # labels cannot be preprocessed so return here
+        if channel.value >= 50:
+            return image
```

Comment:
Hmmm - I don't like this solution. Why not check the channel name?
e.g.:
```python
if channel in (Channels.MASK, Channels.WEIGHTS,)
```

or 
```python
if channel.name.startswith(('MASK', 'WEIGHTS'))
```